### ISSUE TITLE
UI require full redesign

### ISSUE THREAD

**Issue by TheShuEd** [crystallpunk-14/crystall-punk-14#203]
- Standard blue theme to futuristic for fantasy
- The current rework (unfinished) in beige doesn't look good enough color-wise
- More contrast:
![image](https://github.com/crystallpunk-14/crystall-punk-14/assets/96445749/b57c9cd9-a1bd-4e00-966d-0d481b0c9827)
- Slot icons should be more contrasting (accessibility) #67
